site/docs/tools/cli.md

```diff
@@ -158,38 +187,64 @@ Usage: nessie log [OPTIONS] [REVISION_RANGE] [PATHS]...
 
   Show commit log.
 
-  REVISION_RANGE optional branch, tag or hash to start viewing log from. If
-  of the form <hash>..<hash> only show log for given range
+  REVISION_RANGE optional hash to start viewing log from. If of the form
+  <start_hash>..<end_hash> only show log for given range on the particular ref
+  that was provided
 
-  PATHS optional list of paths. If given, only show commits which affected
-  the given paths
+  PATHS optional list of paths. If given, only show commits which affected the
+  given paths
 
 Options:
-  -n, --number INTEGER    number of log entries to return
-  --since, --after TEXT   Commits more recent than specific date
-  --until, --before TEXT  Commits older than specific date
-  --author, --committer   limit commits to specific committer
-  --help                  Show this message and exit.
+  -r, --ref TEXT                  branch to list from. If not supplied the
+                                  default branch from config is used
+  -n, --number INTEGER            number of log entries to return
+  --since, --after TEXT           Only include commits newer than specific
+                                  date
+  --until, --before TEXT          Only include commits older than specific
+                                  date
+  --author TEXT                   Limit commits to a specific author (this is
+                                  the original committer). Supports specifying
+                                  multiple authors to filter by.
+  --committer TEXT                Limit commits to a specific committer (this
+                                  is the logged in user/account who performed
+                                  the commit). Supports specifying multiple
+                                  committers to filter by.
+  --query, --query-expression TEXT
+                                  Allows advanced filtering using the Common
```

Comment:
The CEL references are good - I like it. It's just hard to read here with the small column size. Can we widen the column at least and add some newlines to the message? Or maybe a section/note about CEL on the website and link to that page from the pynesse CLI help?